### PR TITLE
Better Testing Guidelines

### PR DESCRIPTION
**Pull request by ahmadawais**
Quite a few folks have had been wondering [how to test](https://make.wordpress.org/core/2017/06/12/whats-new-in-gutenberg-june-12th/#comment-32701) Gutenberg Editor after installing and building. I add a tip.

### CODE REVIEW

**Comment by aduth** [comment]
Should we adapt the [`TESTS.md` file](https://github.com/ahmadawais/gutenberg/blob/6162eb8a98b64be23f3fda7f36204ec1f152280a/TESTS.md) for this purpose?

`CONTRIBUTING.md` is linked to both when creating a new issue or a new pull request. To avoid an overwhelming blob of text (perhaps evidenced by the unique quote styling you'd applied to the snippet), we might consider splitting messaging between code-focused contributions and usability-testing contributions by one of:

- Better sectioning within `CONTRIBUTING.md`
- References from `CONTRIBUTING.md` to separate specific documents
- Creating an [issue template](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) with helpful information

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ If you have a local WordPress environment, you can clone this repository into yo
 
 Some good options for a local WordPress development environment include <a href="https://varyingvagrantvagrants.org/">VVV</a> and <a href="https://www.mamp.info/">Mamp</a>.
 
+> Once the plugin is intalled, and you have built it through `npm` go to the post list and hover over a post row to click `Edit in Gutenberg` link. That should do it for initial testing.
```
Comment (aduth) [inline CONTRIBUTING.md]: Should we mention the admin bar menu which includes both a Demo and New Post submenus?

Nitpicking, but the last sentence doesn't add much value to the text, and could be dropped without sacrificing clarity.

Comment (ahmadawais) [inline CONTRIBUTING.md]: @aduth I think the admin bar menu gets ignored, pretty easily. But feel free to edit the text before you merge this one. Or let me know if I should do it.

**Comment by ahmadawais** [comment]
@aduth I just pushed an update to the text. Take a look.

**Comment by aduth** [comment]
What do you think about my questions in https://github.com/WordPress/gutenberg/pull/1174#issuecomment-308819903 ? Trying to decide where best this belongs.

Also, with plugin release imminent, might we want to encourage usability testing through the plugin instead? We'd still want to provide pointers to where they'll find the editor in the admin, but they'll not be as relevant to clone/build instructions.

**Comment by gziolo** [comment]
I think it would be good idea to add this text to `TESTS.md` and reference it from `CONTRIBUTING.md`.

**Comment by karmatosed** [comment]
+1 to using TESTS.md for this so closing this PR.

**Pull request closed without merging**